Dockerfile

```diff
@@ -1,26 +1,16 @@
-FROM flink:1.5.1
+FROM alpine:3.8
 
 ENV HIGH_AVAILABILITY=none \
```

Comment:
Do we still need these env vars?

Reply:
goed punt, alleen RPC address/port lijkt me